lib/mirage.ml

```diff
@@ -1566,10 +1566,22 @@ let get_mode () = Key.(get (get_base_context ()) target)
 
 let libraries_ref = ref []
 let add_to_ocamlfind_libraries l =
+  let pp_v fmt lib = Format.fprintf fmt "\"%s\" ;" lib in
+  let list_pp fmt l = Format.pp_print_list pp_v fmt l in
+  Format.eprintf "add_to_ocamlfind_libraries [%a] is deprecated. \
+      Please instead call `register` with `~libraries:[%a]`.\n"
+      list_pp l list_pp l
+  ;
```

Comment:
Hm, it looks to me like I can only give `[@@ocaml.deprecated]` a static string, instead of the error message here which lists the elements in question.